### PR TITLE
Conversation engine in demos

### PR DESCRIPTION
**Pull request by LoyVanBeek**
Depends on https://github.com/tue-robotics/robot_launch_files/pull/28

### CODE REVIEW

File: launch/state_machines/demo_extern.launch [hunk 1]
```diff
@@ -27,8 +27,26 @@
...
+        <include file="$(find robot_launch_files)/launch/interaction/telegram.launch">
+            <arg name="machine" value="sergio2"/>
+            <param name="robot_name" value="$(arg robot_name)" />
+        </include>
+
     </group>
 
+    <!--Start conversation engine-->
+    <node name="conversation_engine" pkg="conversation_engine" type="engine" output="log" respawn="true" args="--robot sergio">
+        <remap from="robot_to_user" to="/sergio/message_to_telegram" />
```
Comment (MatthijsBurgh) [inline launch/state_machines/demo_extern.launch]: If we have both telegram and slack, why use telegram in topics used by both?

**Review by MatthijsBurgh — CHANGES_REQUESTED** [review]

File: launch/state_machines/demo_extern.launch (hunk 1 above)
Comment (LoyVanBeek) [reply launch/state_machines/demo_extern.launch]: Good point. it's just what was made in the German Open but should indeed be renamed. Suggestions for a good name?

Comment (MatthijsBurgh) [reply launch/state_machines/demo_extern.launch]: Maybe:
user_to_converstation
converstation_to_user

Comment (LoyVanBeek) [reply launch/state_machines/demo_extern.launch]: I've named it to ```message_to_user``` and ```message_from_user```

**Comment by LoyVanBeek** [comment]
Tested on robot, works up until calling the action server: send_goal_async of action_server was missing for some reason.

**Review by jlunenburg — CHANGES_REQUESTED** [review]
* Conversation engine, slack bridge and telegram bridge are not included as dependencies in tue-env. Therefore, this will lead to problems after installing a new PC
* There's no clear distinction between sim and real robot. This will lead to issues. IMHO, the conversation engine can run in simulation as well but the bots are not essential (when developing you can use the topics on your laptop anyway)

**Pull request closed without merging**